cmd/cobra/commands/accesstoken.go

```diff
@@ -0,0 +1,108 @@
+package commands
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	jww "github.com/spf13/jwalterweatherman"
+)
+
+var accessTokenCreatCmd = &cobra.Command{
```

Comment:
The current convention is name a variable by its command accordingly, e.g., `create-access-token` -> `createAccessTokenCmd`.